warehouses/fase/ingest/clone_postgresql_table.py

```
@@ -0,0 +1,95 @@
+import json
+import argparse
+import sys
+from sqlalchemy import create_engine, inspect
+from elt_common.dlt_destinations import pyiceberg
+
+BASE_URI = "postgresql://duouser:duopassword@localhost:5432/duo"
+
+def map_pg_to_dlt_type(pg_type):
+    """Maps SQLAlchemy types to dlt core types."""
+    t = str(pg_type).lower()
+    if 'int' in t: return 'bigint'
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)